QHotkey/qhotkey_mac.cpp

```diff
@@ -3,11 +3,17 @@
 #include <Carbon/Carbon.h>
 #include <QDebug>
 
+#if QT_VERSION >= QT_VERSION_CHECK(6, 0, 0)
```

Comment:
Maybe we could move it to `qhotkey_p.h` to avoid duplication?

Reply:
Fine.

The same definitions can also be removed from qhotkey_x11.cpp in favor of the private header. I'll push it.

Reply:
Great! Thank you!